Drag and drop to upload

jbidoret opened the pull request:
Add support for Drag and drop files within the media panel.
A progressive enhancement that triggers the regular file upload field, by filling it with dropped files.

n-peugnet commented on src/media.js:
```diff
@@ -4,3 +4,34 @@ window.addEventListener('load', () => {
+const fileInput = document.getElementById('file');
+const fileMenu = fileInput.closest('details');
+
+'drag dragstart dragend dragover dragenter dragleave drop'.split(' ').forEach(function(listener){
```
I think I would prefer a simpler `for...of` construct. More like:

```js
const dragListeners = ['drag', 'dragstart', 'dragend', 'dragover', 'dragenter', 'dragleave', 'drop']
for (const listener of dragListeners) {
    // register listener
}
```

jbidoret replied:
Sure. Sorry for my js style :)

n-peugnet commented on src/media.js:
```diff
@@ -4,3 +4,34 @@ window.addEventListener('load', () => {
+    },false);
+});
+
+'dragover dragenter'.split(' ').forEach(function(listener){      
```
Same comment here.

n-peugnet reviewed (commented):
Just a small comment

jbidoret commented:
OK. It seems this last commit could pass the tests.
I added multiple drops ability, and modified the dirty string split to a cleaner `for of` loop.
Could you test it and tell me? (no urge!)

vincent-peugnet commented:
> OK. It seems this last commit could pass the tests. I added multiple drops ability, and modified the dirty string split to a cleaner `for of` loop. Could you test it and tell me? (no urge!)

I did some test on my side and it works fine ! Thanks Julien !

I did tests on:

- Firefox
- Chromium

vincent-peugnet commented:
For me it's good to go ! If nobody have any objection I'm going to merge it soon 😃
